sdk-core/src/main/java/com/reach5/identity/sdk/core/ReachFive.kt

```diff
@@ -501,13 +505,51 @@ class ReachFive (
             ))
     }
 
+    fun loginWithWebAuthn(
+        loginRequest: WebAuthnLoginRequest,
+        loginRequestCode: Int,
+        failure: Failure<ReachFiveError>
+    ) =
+        reachFiveApi
+            .createWebAuthnAuthenticationOptions(WebAuthnLoginRequest.enrichWithClientId(loginRequest, sdkConfig.clientId))
+            .enqueue(ReachFiveApiCallback(
+                success = {
+                    val fido2ApiClient = Fido.getFido2ApiClient(activity)
+                    val fido2PendingIntentTask = fido2ApiClient.getSignPendingIntent(it.toFido2Model())
+                    fido2PendingIntentTask.addOnSuccessListener { fido2PendingIntent ->
+                        if (fido2PendingIntent != null) {
+                            try {
+                                Log.d(TAG, "Launching Fido2 Pending Intent")
+                                activity.startIntentSenderForResult(fido2PendingIntent.intentSender, loginRequestCode, null, 0, 0, 0)
+                            } catch (e: IntentSender.SendIntentException) {
+                                e.printStackTrace()
```

Comment:
intended?

Reply:
Sorry, what do you mean? Indentation?

Reply:
We need to call the `printStackTrace` function here?